components/InterestZoneView/UnsterilizedCatList.tsx

```diff
@@ -13,14 +17,22 @@ const UnsterilizedCatsList = (props: Props) => {
     };
 
     return (
-        <form className={styles.catListItem}>
-            <label>Gender</label>
-            <input id="unsterilized-cat-gender-input"  type="text" defaultValue={props.cat?.gender} readOnly={!props.isEditable}></input>
-            <label>Voluntar</label>
-            <input id="unsterilized-cat-volunteer-input" type="text" defaultValue={props.cat?.mediaLinks} readOnly={!props.isEditable}></input>
-            <label>Observatii</label>
-            <input id="unsterilized-cat-observations-input" type="text" defaultValue={props.cat?.observations} readOnly={!props.isEditable}></input>
-        </form> 
+        <Box>
+            <Group sx={{width: '100%'}}>
+                {props.cat?.gender === 'male' && <FontAwesomeIcon icon={faMars}/> }
```

Comment:
if we already have an enum defined with the 2 genders 'male' and 'female', please use that one. If not, it needs to be created, so that we don't have conditional clauses that use locally defined string constants (for reusability purpose)